src/BacktraceFactory.php

```diff
@@ -90,11 +90,28 @@ private function formatBacktrace(array $backtrace) : array
 
             return array_merge($context, [
                 'method' => $frame['function'] ?? null,
-                'args' => $frame['args'] ?? null,
```

Comment:
Not sure if it matters—does the null value behave the same way in the new method?

Reply:
No, it doesn't. Now it throws:
```
"Undefined index: args", "/vendor/honeybadger-io/honeybadger-php/src/BacktraceFactory.php", 93
```
Found during Laravel Forge deployments since updating to honeybadger-laravel 1.7.2 
Ref: Faults 54680936 and 54680937
/cc @sixlive

Reply:
I'll push a patch today, could you open an issue for it?

Reply:
#90 opened.